rules-reviewed/eap7/eap5/jboss-eap5-7-xml.windup.xml

```diff
@@ -160,19 +160,16 @@
                     <link title="JBoss EAP 5 - Login Modules" href="https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html-single/security_guide/#idm139921681412720"/>
                     <tag>security</tag>
                 </classification>
-                <hint title="JBoss security configuration descriptor (prior to AS 7 / EAP 6)" effort="5" category-id="mandatory">
-                    <message>
-                        When migrating to JBoss EAP 6+,
-                        the `login-config.xml` descriptor is no longer supported.
-                        Security is now configured in the `security-domain` element inside the server configuration.
+                <hint title="JBoss security configuration descriptor (login-config.xml)" effort="5" category-id="mandatory">
+                    <message>Before JBoss EAP 6, authentication security domains and login modules could be configured in a `login-config.xml` file.
+                    JBoss EAP 6+ does not support the `login-config.xml` descriptor. Security is now configured inside the server configuration. Please refer to the corresponding server security guide.
                     </message>
-                    <link title="JBoss EAP 7 - How To Configure Server Security" href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0/how-to-configure-server-security/how-to-configure-server-security"/>
+                    <link title="JBoss EAP 6.4 - How To Configure Server Security" href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/version-6.4/red-hat-jboss-enterprise-application-platform-64-how-to-configure-server-security"/>
```

Comment:
We should update to 7.1 documentation?
https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/how_to_configure_server_security/

Reply:
Fixed.